### DISCUSSION TITLE
Firestore Documentation

### DISCUSSION THREAD

**Discussion by kksingla** [kreait/firebase-php#542] [Q&A]
How to use firebase PHP admin SDK for read-write operations?

**Comment by jeromegamez** [comment]
As written on https://firebase-php.readthedocs.io/en/5.14.1/cloud-firestore.html, this SDK provides a bridge to the [google/cloud-firestore](https://packagist.org/packages/google/cloud-firestore) package. The page also has links to the [API docs](https://googleapis.github.io/google-cloud-php/#/docs/cloud-firestore) of the package and an [examples repository.](https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/firestore)
